lib/cog_api/http/groups.ex

```diff
@@ -16,6 +16,11 @@ defmodule CogApi.HTTP.Groups do
     |> format_group_response
   end
 
+  def find(%Endpoint{}=endpoint, filter) do
```

Comment:
Would you mind adding this to the fake so that the interfaces offer the same format?

Also, should this be added to `client.ex`?